pkg/registrar/map_functions.go

```diff
@@ -391,24 +391,22 @@ func (mapper *Mapper) ServiceAccountToWorkloadRequests(serviceAccountObject clie
 	for _, sc := range supplyChains {
 		scWorkloads, err := mapper.clusterSupplyChainToWorkloads(sc)
 		if err != nil {
-			mapper.Logger.Error(err, "service account to workload requests")
+			mapper.Logger.Error(err, fmt.Sprintf("service account to workload requests: cluster supply chain [%s] to workloads", sc.Name))
```

Comment:
Please use [keys and values, not fmt (link)](https://github.com/vmware-tanzu/cartographer/blob/main/CONTRIBUTING.md#best-practices)

Reply:
I was trying to maintain consistency with how we handle errors in this file elsewhere (see [here](https://github.com/vmware-tanzu/cartographer/blob/79aaec9274dce66f8556d818f6bc191a8775fa08/pkg/registrar/map_functions.go#L76), for example).

There is an open issue (#383) that should address this problem in the entire file so I think it would be best to maintain consistency for now and fix it as part of that issue.